docs/gulp.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Gulp
+tags:
+  - gulp
+  - node
+  - sass
+  - js
+  - javascript
+  - scss
+  - frontend
+---
+# Gulp
+
+This project uses [Gulp][] for frontend tooling. The `gulpfile.js` included with
+this project has the following commands:
+
+* `lando gulp build[:js|sass]`
+* `lando gulp watch[:js|sass]`
+* `lando gulp validate[:js|sass]`
+* `lando gulp fix[:js|sass]`
+
+## Build command
+
+The `build` command compiles all SCSS and JavaScript (ES6) files in the project.
+The compiler recursively searches the following directories to compile files:
+
+* `/web/modules/custom`
+* `/web/themes/custom`
+
+**Note:** Files with the extensions `.css` and `.js` will be ignored by git.
+Compiled assets should not exist in version control.
+
+### SCSS compilation
+
+The `build:sass` command will compile files with the extension `.scss` that do
+**not** begin with an underscore (`_`) to `.css` files in the same directory.
+
+#### Example
+
+The file:
+`/web/themes/custom/my_theme/assets/css/main.scss`
```

Comment:
This example seems weird. `.scss` file in a `css` folder? Is this how projects are expected to be set up? Typically we have a `scss` directory for source files and `css` for compiled.

Reply:
We're doing in directory compilation of assets. I can change it to styles, it's just an example though 🤷‍♂️ 

You also already reviewed these files on another repo. 😂

Reply:
Hmm I think I was thinking this would generate a ton of styles side-by-side but it's really only going to be one.

This should be fine. Nevermind!